_posts/nurahill/2017-05-22-nura-tirtle-hack.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+author: nurahill
+title: "Nura's Turtle Exercise"
+---
+
+Here is the program i am embeding:
+ <iframe src="https://trinket.io/embed/python/0af7db0563" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+ 
+ This assignment was fun. Just like the first one. When I started the assignment I had no idea what i wanted to do. So i googled things for inspiration. I did not know that we had to cite where we got code from if we used someone elses code, but i will do my best to find and it add it somewhere. So basically what I designed was a logo (because thats kind of what it looks like). See image below.
+ 
+ ![Turtle image](https://trinket-user-assets.trinket.io/766ef1ddb2e8fcf44c2522a82c38d12138860fcb-59239d72d1f91c5f706e1ad2.png)
```

Comment:
Very precise image!  The symmetry, colors, and shapes work well together.

You did a lot of copying or others' code in this, and I would've liked to have seen more code written entirely by you to supplement the modified example code.  And, of course, accurately citing your examples is required in the future.